api/src/cli/commands/schema/snapshot.ts

```diff
@@ -7,9 +7,21 @@ import inquirer from 'inquirer';
 import { dump as toYaml } from 'js-yaml';
 import { flushCaches } from '../../../cache';
 
+/**
+ * Will sort the yaml keys by alphabetical order.
+ * If the key is a top level key, it will be sorted by its original order (defined in the topLevelKeys array).
+ */
+function sortYaml(a: any, b: any) {
+	const topLevelKeys = ['version', 'directus', 'collections', 'fields', 'relations'];
```

Comment:
You could read those top level keys from the snapshot object instead of hardcoding them.

Reply:
Just added commit to do that. Thanks!